test/performance/mako/analyzer.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mako
+
+import (
+	"github.com/golang/protobuf/proto"
+	tpb "github.com/google/mako/clients/proto/analyzers/threshold_analyzer_go_proto"
+	mpb "github.com/google/mako/spec/proto/mako_go_proto"
+)
+
+// NewCrossRunConfig returns a config that can be used in ThresholdAnalyzer.
+// By using it, the Analyzer will only fail if there are xx continuous runs that cross the threshold.
+func NewCrossRunConfig(runCount int32, tags ...string) *tpb.CrossRunConfig {
```

Comment:
we aren't passing in any tags from our `sla.go` files...

Reply:
I'm thinking about leaving it as a placeholder in case we want to add tags in the future, or do you want me to remove it now?

Reply:
My point is that if this is what's used to establish the window then I think it's incorrect for at least the load-test, which uses tagging to distinguish multiple data points.

Reply:
Fixed.
Initially I was thinking `Mako` should be able to determine which tag(s) to use for the query, but since we are using a few other tags, like commitID and kube version, it makes sense to let us specify the tag(s).